docs/api/safe-storage.md

```diff
@@ -0,0 +1,34 @@
+# safeStorage
+
+> Allows access to simple encryption and decryption of strings.
+
+Process: [Main](../glossary.md#main-process)
+
+Note that on Mac, access to the system Keychain is required and
+these calls can block the current thread to collect user input.
```

Comment:
@MarshallOfSound is this true? I thought the keychain dialog popped up at startup, not when the call happened.

Reply:
I took this from chromium's OSCrypt class- since this API wraps OSCrypt's functionality I just assumed this was true
(sorry for the green text)
![image](https://user-images.githubusercontent.com/33054982/125357587-a878a380-e31c-11eb-8791-88d1086ac060.png)

Reply:
The keychain is only unlocked once on boot, the key is then cached for the lifetime of the app.  So these safeStorage classes will never trigger the prompt afaik